apps/expo/src/components/AppButton.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import { TouchableOpacity, Text, StyleSheet } from "react-native";
+
+interface ButtonProps {
+  text: string;
+  buttonColor: string;
+  borderRadius?: string;
+  marginTop?: number;
+  marginBottom?: number;
+  marginLeft?: number;
+  marginRight?: number;
+}
+
+const borderRadiusSize = (size: string | undefined) => {
+  let transform;
+  switch (size) {
+    case "medium":
+      transform = 15;
+      break;
+    case "large":
+      transform = 25;
+      break;
+    case "full":
+      transform = 50;
+      break;
+
+    default:
+      transform = 0;
+      break;
+  }
+
+  return transform;
+};
+
+export const AppButton = ({
+  text,
+  buttonColor,
+  borderRadius,
+  marginTop,
+  marginBottom,
+  marginLeft,
+  marginRight,
+}: ButtonProps) => {
+  const buttonStyles = StyleSheet.create({
```

Comment:
Since this component is quite complicated, I'll accept nga stylesheet for now over tailwind but prefer tailwind dyapon.